### PR TITLE
Fixed typos first-timers-only issue

### LINKED ISSUE #7165: Fix typos in documentation for reply-by-email feature
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [$FILENAME]($BRANCH_URL) in the `$REPO` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

### Here's what you need to do:

There are some typos in the file EMAIL.md (https://github.com/publiclab/plots2/blob/master/doc/EMAIL.md). Specifically:

- On line 5: "All the configrations regarding this is to be done" should be changed to "All the configurations regarding this are to be done"
- On line 11: "Logs for mailman can be seen oin Mailman log file" should be changed to "Logs for mailman can be seen in Mailman log file"
- On line 12: "3." should be removed. Not sure why it's there.
- On line 20: "corressponding user-tag is generated" should be changed to "corresponding user-tag is generated"

Please fix these typos. In addition to these, there are some areas where rephrasing would be helpful. Optionally, use your discretion in editing other parts of the file to increase the ease of comprehension for this doc.

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by KateAnn19**
I fixed typos in documentation.

Fixes #7165  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by KateAnn19** [comment]
My Twitter is @McGeek42

**Comment by KateAnn19** [comment]
Also, this is my very first time contributing to a project. I was brought here by the Twitter account first-timers-only. In the **steps** it says to ask for a review. Will you all review and get back with me or will they?

**Comment by KateAnn19** [comment]
Fix typos in documentation for reply-by-email feature  "Fixes #7165" . Is this what you mean? Not totally sure how to link it...if you can tell me how I will.

**Comment by anthony-zhou** [comment]
@KateAnn19 To link the issue, replace the "Fixes #0000" in the PR description with "Fixes #7165"

**Comment by KateAnn19** [comment]
Just completed. Thanks for the help.

File: doc/EMAIL.md [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
- 
 ## Email Settings
 
 Users can modify **Email Settings** by visiting [https://publiclab.org/settings](https://publiclab.org/settings). Settings for each user are saved by using UserTags on their profiles. 
 For each setting, there is an associated tag with it. Below is the table showing associated tags with setting. 
 
-We consider absence of tag as true and presence of tag as false for setting, so whenever a user turn-off a setting, corressponding user-tag is generated and turning setting on again will delete the tag.  
-So, while notifying a user UserTag is checked in different files and user is notified depending on the presence or absence of the tag.
+We consider absence of **tag** as true and presence of **tag** as false for setting, so whenever a user turns **off** a setting, a corresponding user-tag is generated and turning setting **on** again will delete the tag.  
+So, while notifying a user **UserTag** is checked in different files and _user_ is notified depending on the presence or absence of the tag.
```
Comment (anthony-zhou) [inline doc/EMAIL.md]: The italics on `_user_` probably aren't needed here. You could change this to say "the user is notified" so it'd be easier to understand.

File: doc/EMAIL.md [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
 
- 
 ## Email Settings
 
 Users can modify **Email Settings** by visiting [https://publiclab.org/settings](https://publiclab.org/settings). Settings for each user are saved by using UserTags on their profiles. 
 For each setting, there is an associated tag with it. Below is the table showing associated tags with setting. 
 
-We consider absence of tag as true and presence of tag as false for setting, so whenever a user turn-off a setting, corressponding user-tag is generated and turning setting on again will delete the tag.  
-So, while notifying a user UserTag is checked in different files and user is notified depending on the presence or absence of the tag.
+We consider absence of **tag** as true and presence of **tag** as false for setting, so whenever a user turns **off** a setting, a corresponding user-tag is generated and turning setting **on** again will delete the tag.
```
Comment (anthony-zhou) [inline doc/EMAIL.md]: I like the emphasis on "off" and "on", but "tag" probably doesn't need to be bolded here. Also, I think the instances of "tag" that are currently bolded should be changed to "the absence of a tag" and "the presence of a tag" to make it easier to understand.

**Comment by KateAnn19** [comment]
I am happy to make these changes, but how do I make these changes since I already did a pull request?

**Comment by anthony-zhou** [comment]
@KateAnn19 You have to find the same branch you were working on (KateAnn19:patch-1) and make the edits there. Then they will automatically show up here.

**Comment by KateAnn19** [comment]
I'm having a hard time finding patch-1 branch.

**Comment by KateAnn19** [comment]
I was able to find it, and I made the changes. Do I need to make another pull request?

**Comment by Uzay-G** [comment]
No you don't need to @KateAnn19. The changes automatically update and this pull request has your changes :+1:

**Comment by KateAnn19** [comment]
Please let me know if there's anything I need to do to get this completely done.

**Comment by SidharthBansal** [comment]
@Uzay-G @anthony-zhou kindly approve it. I will merge once you both approve
thanks a lot

**Pull request merged**